### ISSUE TITLE
Colour & bold with `--color never`

### ISSUE THREAD

**Issue by Proximyst** [anderejd/cargo-geiger#60]
When ran with `--color never`, the result will include colour codes for the `Scanning...Done` messages (which go to stdout, by the way, unlike the normal behaviour of most cargo features).
The tree also includes bold text, the symbol for unsafe usage includes `^M^[[6C^, `Dependency` ends with `^[[0m`, and all dependencies start with `^M^[[51C`.

**Comment by anderejd** [comment]
Ooh nice find. This should be easy to fix, want to take a stab at it?

**Comment by Proximyst** [comment]
@anderejd I'm trying to figure out how the whole config mess works. How would I find out whether colours should be used? Also how come you do this?
```rust
    let verbosity = if args.verbose == 0 {
        Verbosity::Normal
    } else {
        Verbosity::Verbose
    };
```
The verbosity should be equal to that created when using `Config::configure`.

**Comment by anderejd** [comment]
>  I'm trying to figure out how the whole config mess works. How would I find out whether colours should be used?

I wasn't aware of the `--colors flag` until you pointed it out just now. Taking a quick look in the cargo config code and docs https://docs.rs/cargo/0.37.0/src/cargo/util/config.rs.html it seems like the only way to get the color and verbosity setting out of the config is by using the stringly typed get_string:

**Comment by anderejd** [comment]
Fixed by: #66
